packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+	<div>
+		<!-- <i class="pi pi-magic" /> -->
+		<InputText
```

Comment:
My intentions are to prettify this in the llm notebook task

Reply:
_Famous last words_